code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -306,14 +307,20 @@
 		if(!LAZYLEN(parts))
 			if(undismembermerable_limbs) //they have limbs we can't remove, and no parts we can, attack!
 				return ..()
+			C.Weaken(30)
+			visible_message("<span class='danger'>[src] feints a deathblow against [C], they freeze in terror!</span>")
 			to_chat(src, "<span class='cultlarge'>\"Bring [C.p_them()] to me.\"</span>")
-			return FALSE
-		do_attack_animation(C)
+			do_attack_animation(C)

```

Comment:
this shouldn't be indented, and you need to return FALSE under the if(!LAZYLEN(parts))